examples/src/examples/user-interface/world-ui.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import * as pc from 'playcanvas/build/playcanvas.js';
+import { AssetLoader } from '../../app/helpers/loader';
+import Example from '../../app/example';
+
+class WorldScreenExample extends Example {
+    static CATEGORY = 'User Interface';
+    static NAME = 'World UI';
+
+    load() {
+        return <>
+            <AssetLoader name="checkboard" type="texture" url="static/assets/textures/checkboard.png" />
+            <AssetLoader name='font' type='font' url='static/assets/fonts/courier.json' />
+            <AssetLoader name='script' type='script' url='static/scripts/camera/orbit-camera.js' />
+        </>;
+    }
+
+    example(canvas: HTMLCanvasElement, assets: { checkboard: pc.Asset, font: pc.Asset, script: pc.Asset }): void {
+
+        // Create the application with input and start the update loop
+        const app = new pc.Application(canvas, {
+            mouse: new pc.Mouse(document.body),
+            touch: new pc.TouchDevice(document.body),
+            elementInput: new pc.ElementInput(canvas)
+        });
+        app.start();
+
+        // Create an Entity with a camera component and simple orbiter script
+        const camera = new pc.Entity();
+        camera.addComponent("camera", {
+            clearColor: new pc.Color(30 / 255, 30 / 255, 30 / 255)
+        });
+        camera.rotateLocal(-30, 0, 0);
+        camera.translateLocal(0, 0, 7);
+        camera.addComponent("script");
+        camera.script.create("orbitCamera", {
+            attributes: {
+                inertiaFactor: 0.2 // Override default of 0 (no inertia)
+            }
+        });
+        camera.script.create("orbitCameraInputMouse");
+        camera.script.create("orbitCameraInputTouch");
+        app.root.addChild(camera);
+
+        // Create an Entity for the ground
+        const material = new pc.StandardMaterial();
+        material.diffuse = pc.Color.WHITE;
+        material.diffuseMap = assets.checkboard.resource;
+        material.diffuseMapTiling = new pc.Vec2(50, 50);
+        material.update();
+
+        const ground = new pc.Entity();
+        ground.addComponent("render", {
+            type: "box",
+            material: material
+        });
+        ground.setLocalScale(50, 1, 50);
+        ground.setLocalPosition(0, -0.5, 0);
+        app.root.addChild(ground);
+
+        // Create an entity with a light component
+        const light = new pc.Entity();
```

Comment:
is the light needed in this example? I'm not sure UI uses lights

Reply:
The UI uses emissive materials, so indeed the light is not used. But this example has a `render` component with a box and a texture to represent the world.